src/oss/confluent.sh

```diff
@@ -1674,11 +1713,14 @@ These are the available commands:
 
     acl         Specify acl for a service.
     config      Configure a connector.
+    consume     Use the kafkacat utility to consume from topics
     current     Get the path of the data and logs of the services managed by the current confluent run.
     destroy     Delete the data and logs of the current confluent run.
+    kafkacat    Use the kafkacat utility to produce to and consume from topics
     list        List available services.
     load        Load a connector.
     log         Read or tail the log of a service.
+    produce     Use the kafkacat utility to produce to topics
     start       Start all services or a specific service along with its dependencies
```

Comment:
Is there a cluster and topic metadata list mode? (e.g., kafka-topics --describe or kafkacat -L)
It is useful for finding partition states and errors (e.g., insufficient ISRs, no leader, whatnot)

Reply:
We discussed a bit in slack, but my take is that there are a lot of other things we could expose at a top level (metadata listing mode, group consumer, query tools, etc).  But then that would just be wrapping more things around kafkacat which is not the immediate goal.  The goal is to work our way towards the converged CLI.  We already know that converged CLI have `confluent produce` and `confluent consume`, so it seems safe to introduce those at the top level.  We don’t know how the converged CLI will deal with metadata listings, etc, so I’m hesitant to make those decisions now since we have realistically no time before preview cutoff

Reply:
I agree with this comment from @edenhill 
Seems that it'd make sense to also add a basic `topic` command (at least for inspection).